### DISCUSSION TITLE
Dash playing back wrong stream

### DISCUSSION THREAD

**Discussion by BluchipStudio** [streamlink/streamlink#4606] [Q&A]
I am trying to play a dash manifest using streamlink but when i do it seems to playback a random stream from the past

I have tested the same stream in vlc and it plays back correctly the stream in question uses the $number$ based segment ids.

I can only guess streamlink is incorrectly calculating the number for the current segment compared to vls how does streamlink calculate these values ?

**Comment by bastimeyer** [comment]
Please open a proper bug report on the issue tracker. The discussion forum is not meant for bug reports.
